### PR TITLE
https://issues.redhat.com/browse/ACM-15644

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-15644

### CODE REVIEW

**Review by JustinKuli — COMMENTED** [review]
Thanks for getting these! I found one more spot I think it's mentioned: https://github.com/stolostron/rhacm-docs/blob/2.13_stage/governance/grc_intro.adoc

**Review by dockerymick — COMMENTED** [review]
@JustinKuli thanks for catching the other link reference from the [Governance](https://github.com/stolostron/rhacm-docs/blob/2.13_stage/governance/grc_intro.adoc) page.  @jc-berger Another thing to consider in the future is making sure we do not link to Policy compliance API from the console. I checked the following file: https://github.com/stolostron/console/blob/release-2.13/frontend/src/lib/doc-util.tsx

I confirmed that there are no links to in the console by using the find option. I looked up the following anchors: `compliance-history-api` and `configure-compliance-history`

File: governance/grc_intro.adoc [hunk 1]
```diff
@@ -8,7 +8,6 @@ Continue reading the related topics of the {acm-short} governance framework:
 * xref:../governance/policy_deployment.adoc#policy-deployment[Policy deployment]
 * xref:../governance/policy_controllers_intro.adoc#policy-controllers[Policy controllers]
 * xref:../governance/policy_ctrl_adv_config.adoc#policy-controller-advanced-config[Policy controller advanced configuration]
-* xref:../governance/compliance_history.adoc#configure-compliance-history[Configuring policy compliance history API (Technology Preview)]
```
Comment (dockerymick) [inline governance/grc_intro.adoc:11]: Reminder to remove this file from the governance/main.adoc file

File: governance/compliance_history.adoc [hunk 2]
```diff
@@ -290,5 +290,4 @@ oc -n open-cluster-management edit AddOnDeploymentConfig governance-policy-frame
 [#add-compliance-history-resources]
 == Additional resource
 
-* See link:../apis/compliancehistory.json.adoc[Policy compliance history API (Technology Preview)].
 * See xref:../governance/hub_policy_framework.adoc#hub-policy-framework[Hub cluster policy framework]
```
Comment (dockerymick) [inline governance/compliance_history.adoc]: governance/compliance_history.adoc file should be removed

**Review by JustinKuli — APPROVED** [review]

File: governance/hub_policy_framework.adoc [hunk 3]
```diff
@@ -282,5 +282,4 @@ Continue reading the related topics of the {acm-short} governance framework:
 - xref:../governance/grc_dashboard.adoc#grc-dashboard[Governance dashboard]
 - xref:../governance/manage_policies.adoc#managing-security-policies[Managing security policies]
 - xref:../governance/policy_dependencies.adoc#policy-dependencies[Policy dependencies]
-- xref:../governance/compliance_history#configure-compliance-history[Configuring policy compliance history (Technology Preview)]
 - Return to the xref:../governance/policy_deployment.adoc#policy-deployment[Policy deployment]
```
Comment (dockerymick) [inline governance/hub_policy_framework.adoc]: This suggestion is just part of improving the governance doc 
```suggestion
- See xref:../governance/policy_deployment.adoc#policy-deployment[Policy deployment] for more information.
```

**Review by dockerymick — APPROVED** [review]
looks good to me!

**Pull request merged**